R/meetup_query.R

```diff
@@ -0,0 +1,26 @@
+
+#' General Meetup query
+#'
+#' Create your own query to the Meetup API
+#' (https://www.meetup.com/meetup_api/docs/).
+#'
+#'
+#' @param api_path url path to send the meetup query after the 'https://api.meetup.com/'
+#' @param ... additional parameters to the query
+#'
+#' @return raw list results from the query, untitied
```

Comment:
```suggestion
#' @return raw list results from the query, untidied
```

Is untidied actually correct? :thinking: